fractal_server/app/models/security.py

```diff
@@ -93,24 +57,59 @@ class UserOAuth(SQLModel, table=True):
     is_superuser: bool = Field(False, nullable=False)
     is_verified: bool = Field(False, nullable=False)
 
-    username: Optional[str]
+    username: str | None
 
     oauth_accounts: list["OAuthAccount"] = Relationship(
         back_populates="user",
         sa_relationship_kwargs={"lazy": "joined", "cascade": "all, delete"},
     )
 
-    user_settings_id: Optional[int] = Field(
+    user_settings_id: int | None = Field(
         foreign_key="user_settings.id", default=None
     )
-    settings: Optional[UserSettings] = Relationship(
+    settings: UserSettings | None = Relationship(
         sa_relationship_kwargs=dict(lazy="selectin", cascade="all, delete")
     )
 
     class Config:
         orm_mode = True
 
 
+class OAuthAccount(SQLModel, table=True):
```

Comment:
Because we add `user: Optional["UserOAuth"] = Relationship(back_populates="oauth_accounts")`, and `"UserOAuth" | None` is invalid (we must use something like `"UserOAuth | None"`.

Tuned out we didn't need ` Optional["UserOAuth"]` in the first place, because ` UserOAuth` is defined in the same module. Moving the definition of `OAuthAccount` below allows us to use `user: UserOAuth | None = Relationship(...)`.